Public/JS/style.js

```diff
@@ -16,6 +16,24 @@
     const slids  [ocuerySelectorAll('.slider__body')];
 >>>>>>> 9ad5bda3b1ce3dbafcf0a58acd19bb2b989729be
 
+<<<<<<< HEAD
```

Comment:
Codacy has a fix for the issue: [Replace `·········value·=·value===0·?·slidersr.length:` with `······value·=·value·===·0·?·slidersr.length·:·`](https://app.codacy.com/gh/sanchezabad034/CALIDADSOFTWARE/pullRequest?prid=9667148)
```suggestion
      value = value === 0 ? slidersr.length : 1;
```